ignite/cmd/scaffold_type.go

```diff
@@ -9,8 +11,10 @@ import (
 // NewScaffoldType returns a new command to scaffold a type.
 func NewScaffoldType() *cobra.Command {
 	c := &cobra.Command{
-		Use:     "type NAME [field]...",
+		Use:     "type NAME [field:type] ...",
 		Short:   "Type definition",
+		Long:    fmt.Sprintf("Type information\n%s\n", supportFieldTypes),
+		Example: "   ignite scaffold type todo-item priority:int desc:string tags:array.string done:bool",
```

Comment:
nit, spacing.

Reply:
That spacing is there to align with "Flags:" and "Usage:" indentation I think. Example will output the text starting from column 0.